images/.github/.hintrc

```diff
@@ -0,0 +1,18 @@
+{
+  "connector": {
+    "name": "local",
+    "options": {
+      "pattern": ["**", "!.git/**", "!node_modules/**"]
+    }
+  },
+  "extends": ["development"],
+  "formatters": ["stylish"],
+  "hints": [
+    "button-type",
+    "disown-opener",
+    "html-checker",
+    "meta-charset-utf-8",
+    "meta-viewport",
+    "no-inline-styles:error"
+  ]
+}
```

Comment:
- [ ] Kindly I recommend you to remove the duplicated `.hintrc` and `.stylelintrc.json` file under ".github" folder inside "images" folder it makes your code base ambiguous and unclean.